collect_app/src/main/java/org/odk/collect/android/activities/GeoPolyActivity.java

```diff
@@ -341,11 +341,13 @@ private String formatPoints(List<MapPoint> points) {
         StringBuilder result = new StringBuilder();
         for (MapPoint point : points) {
             // TODO(ping): Remove excess precision when we're ready for the output to change.
-            result.append(String.format(Locale.US, "%s %s %s %s;",
+            result.append(String.format(Locale.US, "%s %s %s %s; ",
```

Comment:
Please make sure your PR description is really specific and describes changes to output data in detail. There's huge risk in making these kinds of changes because they can break all downstream analysis.

I think this change comes from a misunderstanding of https://github.com/getodk/collect/issues/4179. There are NOT supposed to be spaces between points in shapes and traces. A user has reported that there are sometimes spaces but we haven't yet been able to reproduce. If you know of a place where spaces are introduced, that would be helpful. This change must be reverted.

Reply:
Oh alright @lognaturel, I'll revert this change. Yes that was a misunderstanding. Also, I noticed this on the form: 
[4228AllWidgetsWithAnswersReadOnly.xml.txt](https://github.com/getodk/collect/files/6143298/4228AllWidgetsWithAnswersReadOnly.xml.txt)

![Screenshot_20210315_225542](https://user-images.githubusercontent.com/35730054/111194985-c5f16700-85e1-11eb-95f0-dce30a9267ba.png)

Reply:
There can be a difference between the underlying value and the display value, that's ok. It does feel to me like a display value with spaces is being saved somehow but we'll need to reproduce to fix.